service/src/main/java/uk/gov/hmcts/reform/fpl/service/sdo/StandardDirectionsOrderService.java

```diff
@@ -81,9 +81,9 @@ public StandardDirectionOrder buildOrderFromUpload(StandardDirectionOrder curren
             .uploader(userInfo.getName())
             .orderDoc(prepareOrderDocument(currentOrder.getOrderDoc(), currentOrder.getOrderStatus()));
 
-        if (featureToggleService.isSendNoticeOfProceedingsFromSdo()) {
-            builder.judgeAndLegalAdvisor(currentOrder.getJudgeAndLegalAdvisor());
-        }
+
+        builder.judgeAndLegalAdvisor(currentOrder.getJudgeAndLegalAdvisor());
```

Comment:
you can chain this one with previous statement

 return  StandardDirectionOrder.builder()
            .orderStatus(currentOrder.getOrderStatus())
            .dateOfUpload(time.now().toLocalDate())
            .uploader(userInfo.getName())
            .orderDoc(prepareOrderDocument(currentOrder.getOrderDoc(), currentOrder.getOrderStatus()))
            .judgeAndLegalAdvisor(currentOrder.getJudgeAndLegalAdvisor())
            .build();